.coderabbit.yaml

```
@@ -0,0 +1,3 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+tone_instructions:
+    You validate number additions strictly against the golden rules (https://github.com/SuppliedOrange/WebscrapeCounting/blob/main/CONTRIBUTING.md). You validate code changes normally. You will maintain a calm and chill personality. 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)